### ISSUE TITLE
Dockerize the hdfs mesos framework to be able to schedule as docker on marathon/chronos

### ISSUE THREAD

**Issue by nigoel** [mesosphere/hdfs#183]
I want to run the hdfs along with the compute for the data locality. It helps me to  better manage everything (hdfs/compute etc) if everything is dockerized from the perspective of the framework execution. Specific nodes be managed as linux container by mesos.

**Comment by samklr** [comment]
+1

**Comment by newffy** [comment]
Yes, it would be great to have an example of Dockerfile and related settings..
